services/order.py

```diff
@@ -0,0 +1,34 @@
+from django.contrib.auth import get_user_model
+
+from django.db import transaction
+from django.db.models import QuerySet
+
+from db.models import Order, Ticket
+
+
+def create_order(
+        tickets: list[dict],
+        username: str,
+        date: str | None = None
+) -> Order:
+    with transaction.atomic():
+        user = get_user_model().objects.get(username=username)
+        order = Order.objects.create(user=user)
+        if date:
+            order.created_at = date
+            order.save()
+
+        for ticket in tickets:
+            Ticket.objects.create(
+                movie_session_id=ticket["movie_session"],
+                order=order,
+                row=ticket["row"],
+                seat=ticket["seat"]
+            )
+
+        return order
+
+
+def get_orders(username: str | None = None) -> QuerySet:
+    return get_user_model().objects.get(username=username).orders.all() \
```

Comment:
1. remember - we don't use `\` for breaking lines as per code style
2. Here you can try different approach: get order queryset, if username is present - filter queryset using lookup on user field, then return queryset